datafusion/core/src/physical_plan/functions.rs

```diff
@@ -52,6 +52,200 @@ use datafusion_physical_expr::math_expressions;
 use datafusion_physical_expr::string_expressions;
 use std::sync::Arc;
 
+macro_rules! make_utf8_to_return_type {
```

Comment:
Seems to be some commit-related problems

Reply:
Fixed that